File: C/pE_7.c
//project euler problem 7

#include <stdio.h>

int main(){
	int s = 0;
	for (int i=2; i<1000000000; i++){
		int a = 0;
		for (int j=2; j<i; j++){
			if (i%j == 0){
				a += 1;
				break;}
		}
		if (a == 0)
			s += 1;
		
		if (s== 10001){
			printf("%d \n", i);
			break;
		}
	}
}

File: PROJECT_EULER_SOLUTIONS/Python/pE_7.py
# project euler problem 7

import sys

s=0
for i in range(2, sys.maxsize):
    a = 0 
    for j in range(2,int(i**0.5) + 1):
        if (i%j == 0):
             a += 1
             break
    if a ==0:
        s += 1
    if s == 10001:
        print('answer is ',i)
        break

